Eatery/Controllers/Eateries/TabbedPageViewController.swift

```diff
@@ -1,145 +1,183 @@
+//
+//  TabbedPageViewController.swift
+//  Eatery
+//
+//  Created by William Ma on 9/25/19.
+//  Copyright © 2019 CUAppDev. All rights reserved.
+//
+
 import UIKit
 
+// MARK: -
+
 protocol TabbedPageViewControllerDelegate: AnyObject {
-    func selectedTabDidChange(_ newIndex: Int)
-}
 
-protocol TabbedPageViewControllerScrollDelegate: AnyObject {
-    func scrollViewDidChange()
+    func tabbedPageViewController(_ tabbedPageViewController: TabbedPageViewController,
+                                  titleForViewController viewController: UIViewController) -> String?
+
+    func tabbedPageViewController(_ tabbedPageViewController: TabbedPageViewController,
+                                  heightOfContentForViewController viewController: UIViewController) -> CGFloat
+
 }
 
-private let kTabBarHeight: CGFloat = 32.0
+// MARK: -
 
-class TabbedPageViewController: UIViewController, UIPageViewControllerDataSource, UIPageViewControllerDelegate, TabBarDelegate {
-    
-    var viewControllers: [UIViewController]!
-    fileprivate var meals: [String] = []
-    
-    weak var tabDelegate: TabbedPageViewControllerDelegate?
-    weak var scrollDelegate: TabbedPageViewControllerScrollDelegate?
-    var tabBar: UnderlineTabBarView?
-    
-    var pageViewController: UIPageViewController!
+class TabbedPageViewController: UIViewController {
+
+    let viewControllers: [UIViewController]
+
+    private var pageViewController: UIPageViewController!
+    private var tabBar: UnderlineTabBarView?
+
+    weak var delegate: TabbedPageViewControllerDelegate?
```

Comment:
can we make `private` if possible?

Reply:
Referencing our previous conversation Apple's frameworks follow this style of delegate and also this pattern is used throughout all of Eatery currently.